src/main/java/org/broadinstitute/hellbender/utils/read/GATKReadToBDGAlignmentRecordConverter.java

```diff
@@ -0,0 +1,34 @@
+package org.broadinstitute.hellbender.utils.read;
+
+import htsjdk.samtools.SAMFileHeader;
+import org.apache.spark.api.java.function.Function;
+import org.bdgenomics.formats.avro.AlignmentRecord;
+import org.bdgenomics.adam.converters.SAMRecordConverter;
+import org.bdgenomics.adam.models.SequenceDictionary;
+import org.bdgenomics.adam.models.RecordGroupDictionary;
+
+
+import java.io.Serializable;
+
+/**
+ * Converts a GATKRead to a BDG AlignmentRecord
+ */
+public class GATKReadToBDGAlignmentRecordConverter implements Function<GATKRead, AlignmentRecord> {
```

Comment:
Sorry, I didn't explain what I wanted very well.

Currently, the converter is a `Function` that's only used in a `.map` call. I think what you're writing will be more generally useful and would be called outside of the context of `.map`.

I was thinking something like this.

```
GATKRead r = ...
r.setSomething();
...
AlignmentRecord ar = GATKReadToBDGAlignmentRecord.convert(r);
ar.something();
```
The map call would then be:
```
myRdd.map(GATKReadToBDGAlignmentRecord::convert)
```